docs/index.md

```diff
@@ -1193,14 +1193,14 @@ A typical setup might look something like the following, where we call `mocha.se
 <head>
   <meta charset="utf-8">
   <title>Mocha Tests</title>
-  <link href="https://cdn.rawgit.com/mochajs/mocha/2.2.5/mocha.css" rel="stylesheet" />
+  <link href="https://unpkg.com/mocha@5.1.1/mocha.css" rel="stylesheet" />
 </head>
 <body>
   <div id="mocha"></div>
 
-  <script src="https://cdn.rawgit.com/jquery/jquery/2.1.4/dist/jquery.min.js"></script>
-  <script src="https://cdn.rawgit.com/Automattic/expect.js/0.3.1/index.js"></script>
-  <script src="https://cdn.rawgit.com/mochajs/mocha/2.2.5/mocha.js"></script>
+  <script src="https://code.jquery.com/jquery-3.3.1.min.js"></script>
+  <script src="http://www.chaijs.com/chai.js"></script>
```

Comment:
maybe `https://unpkg.com/chai@4.1.2/chai.js` instead

Reply:
Think referencing the official site is better personally. Otherwise, leave the version out of Chris's suggested URL (a la `https://unpkg.com/chai/chai.js`) or constantly update the document chasing the current versions of each referenced software.

Reply:
I got the chai URL from [official document](http://www.chaijs.com/guide/installation/). To me, chai version does not matter here.

Reply:
We should use the official one - <http://chaijs.com/chai.js>; shame they don't have it available via HTTPS.

Reply:
`https://unpkg.com/chai/chai.js` is fine; we just want a CDN.